javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,160 @@
+let commnadCounter = 0;
+let userName = "";
+let toDoList = [];
+
+const checkNameStr = "Hello my name is";
+const sayNameStr = "What is my name";
+const toDoListAddStr = " to my todo list";
+const toDoListRemoveStr = " from my todo list";
+const checkAddStr = "Add ";
+const checkRemoveStr = "Remove ";
+const printToDoStr = "What is on my todo?";
+const checkDayStr = "What day is it today?";
+const months = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+const setTimerStr = "Set a timer for ";
+const mathStr = "What is ";
+let firstDigit;
+let secondDigit;
+let checkCounter = "How many commands have I written?";
+
+function getReply(command) {
+  commnadCounter = commnadCounter + 1;
+
+  if (command.indexOf(checkNameStr) >= 0) {
+    if (userName.length == 0) {
+      userName = command.slice(checkNameStr.length + 1, command.length);
+      console.log("Nice to meet you, " + userName + ".");
+    } else {
+      console.log("I already know your name, " + userName + " :)");
+    }
+  }
```

Comment:
really smart! took me a second to understand what you did here, so remember to comment your code here and there :)